### ISSUE TITLE
Contao 3.5.28 ics Kalender (google) wird nicht importiert

### ISSUE THREAD

**Issue by webmurkser** [hschottm/calendar_ical#12]
Leider gibt es keine Fehlermeldung -> weder im /logs/ oder Contao intern. 
Ändere ich ein Einstellung > zB. die Cache Zeit auf 30 Sekunden gibt es im Backend nur folgendes Systemlog:

 
[23.08.2017 13:33] Version 2 of record "tl_calendar.id=2" has been created |  
[23.08.2017 13:33] Reload iCal Web Calendar googletermine (2): Triggered by 1503488038 - 1503487993 = 1503488038 > 30 
[23.08.2017 13:33] reading cal 
[23.08.2017 13:33] Version 1 of record "tl_calendar.id=2" has been created

**Comment by Jaaaan** [comment]
Hast du in den Google Einstellungen mal versucht, den Kalender öffentlich zu machen?
Du könntest auch mal überprüfen, ob der Kalender-Feed von Google überhaupt erstellt wird. Dazu die URL (endet hoffentlich auf .ics) im Browser aufrufen, es sollte dann ein Download angeboten werden.
Falls nicht, kommt eine Google Fehlermeldung (z. B. Kalender-Feed konnte nicht erstellt werden).

**Comment by webmurkser** [comment]
Ja, habe ich - der link ist: https://calendar.google.com/calendar/ical/e16qn1if96osrh1beksfks6rc0%40group.calendar.google.com/public/basic.ics

Es ist ein "Bundesliga" Terminplan - daher so ohne ständiges Editieren der Termine immer aktuell. Wenn man den Link im Browser "öffnet" wird einem die ICS Datei zum speichern angeboten.

**Comment by Jaaaan** [comment]
Bei mir läuft unter Contao 3.5.28 der Import der Termine mit deiner URL einwandfrei. Im Backend-Systemlog sieht es so aus wie bei dir.
Hast du die Erweiterung über die Erweiterungsverwaltung oder mittels Composer (Client) installiert?
Hast du als Zeitzone `CET`?
Läuft der Import denn mit anderen Kalenderadressen?

**Comment by webmurkser** [comment]
Installiert via ER2 und Zeitzone hatte ich auf Europa/Berlin gestellt. 
Ich habe jetzt einen anderen Buli Kalender mit den gleichen Einstellungen (Cache und Zeitzone eingestellt) geladen: http://i.cal.to/ical/3/borussiadortmund/bundesliga-spielplan/327405bb.85b238eb-e37b07ed.ics und dieser hat sofort funktioniert...

Ich hatte gestern bestimmt 3 mal den Kalender gelöscht und neu eingerichtet - also daran kann es eigentlich nicht liegen. Ich nehme das jetzt als kein Bug im calender_ical sondern ein Google-Problem wobei du ja die Einbindung geschafft hast. Merkwürdig.

**Comment by webmurkser** [comment]
Der Kalender lädt die ics Ereignisse und dann mal wieder nicht. Auch wenn sich die ICS Datei (also der Kalender dahinter) gar nicht verändert hat, kann es vorkommen, dass die Ereignisse nach dem Cache-Refresh (also der Zeit die man im Backend angibt) komplett verschwinden. 
Ändert man dann die Cache Zeit auf 60 Sekunden - kann es passieren, dass gleich geladen wird oder erst mehrere Versuche später. 

Dazu im Backend folgendes geloggt:

[07.09.2017 19:18] Reload iCal Web Calendar Termine2017-2018 (3): Triggered by 1504804695 - 1504804515 = 180 > 60 |  
[07.09.2017 19:18] reading cal |  
[07.09.2017 19:15] Reload iCal Web Calendar Termine2017-2018 (3): Triggered by 1504804513 - 1504804378 = 1504804513 > 60 |  
[07.09.2017 19:15] reading cal |  
[07.09.2017 19:12] Reload iCal Web Calendar Termine2017-2018 (3): Triggered by 1504804347 - 1504804173 = 174 > 60 |  
[07.09.2017 19:12] reading cal |  
[07.09.2017 19:09] Reload iCal Web Calendar Termine2017-2018 (3): Triggered by 1504804172 - 1504804076 = 96 > 60 |  
[07.09.2017 19:09] reading cal
